ghost/admin/app/components/gh-nav-menu/footer.js

```diff
@@ -35,6 +63,22 @@ export default class Footer extends Component {
         });
     }
 
+    @action
+    loadCurrentMRR() {
+        this.dashboardStats.loadMrrStats();
+    }
+
+    @action
+    dismissReferralInvite(event) {
+        event.preventDefault();
+        event.stopPropagation();
+        const key = 'referralInviteDismissed';
+        const value = moment().tz(this.settings.timezone);
+        const options = {user: true};
+
+        return this.feature.update(key, value, options);
```

Comment:
`this.feature.update` is more of an internal implementation detail and shouldn't be used outside of the feature service.

Inside the feature service you can add

```js
@feature('referralInviteDismissed', {user: true}) referralInviteDismissed;
```

then in this code you can use that directly, e.g.

```js
if (!this.feature.referralInviteDismissed) {}`

this.feature.referralInviteDismissed = moment().tz(this.settings.timezone)
```